### DISCUSSION TITLE
Explain schedule demo link?

### DISCUSSION THREAD

**Discussion by holgerbrandl** [TimefoldAI/timefold-solver#599] [General]
A while back (still in optaplanner age), there was an article or video I believe, where you had demoed how a user could change a schedule (or some planning solution of some type) interactively via a web-ui, and the explain-API was used to indicate the plan improvement or deterioration down to the task level. Is this article still online? It was a wonderful example of establishing trust and acceptance in an automated scheduling approach.

**Comment by triceo** [comment]
I don't recall having a video like that, but from the description, it sounds like it must have been one of the OptaWeb Employee Rostering videos. (Maybe the COVID-specific hospital scheduling variant we did during the early days of the pandemic?) I know that OptaWeb Employee Rostering had good support for score justifications.

That said, in the modern era of Timefold, we now have a brand new feature - [Score Analysis](https://timefold.ai/docs/timefold-solver/latest/constraints-and-score/understanding-the-score#scoreAnalysis) - which can be seen in the School Timetabling quickstarts.

**Comment by holgerbrandl** [comment]
Found it: https://www.youtube.com/watch?v=4H3U6xyCgMI It was the wonderful presentation "Explanation by example" by Zonca et. al detailing counterfactuals and how to use optplanner as a decision support engine. 

Thanks for your support.
